pandas/tests/io/formats/test_to_csv.py

```diff
@@ -257,6 +257,20 @@ def test_to_csv_multi_index(self):
         exp = "foo\nbar\n1\n"
         assert df.to_csv(index=False) == exp
 
+        # GH 19589
+        idx_1d = pd.MultiIndex(levels=[[1.0]], labels=[[0]], names=['x'])
+        idx_2d = pd.MultiIndex(
+            levels=[[1.], [2.]], labels=[[0], [0]], names=['x', 'y']
+        )
+        expected1d = 'x,data\n1.0,1\n'
+        expected2d = 'x,y,data\n1.0,2.0,1\n'
```

Comment:
use the pattern
```
result = 
expected = 
assert result == expected
```
use a blank line to separate cases

Reply:
Thanks! Indeed it is easier to read in the suggested form. 168833489d22ac79e1e214a70f93096c388c9852